### PR TITLE
NAS-112299 / 12.0 / fix ctld.conf generation when sharing raw disks

### PR DESCRIPTION
**Pull request by yocalebo**
The changes in `12.0-U5.1` to optimize systems with 100's+ of hard drives broke an edge-case where we allow configuration of iSCSI on CORE sharing raw disks. (SCALE does not allow this, thankfully).

### CODE REVIEW

**Review by rick-mesta — APPROVED** [review]

File: src/middlewared/middlewared/etc_files/ctld.py [hunk 1]
```diff
@@ -212,7 +213,7 @@ def main(middleware):
     poolthreshold = {}
     zpoollist = {i['name']: i for i in middleware.call_sync('zfs.pool.query')}
 
-    system_disks = middleware.call_sync('device.get_disks')
```
Comment (sonicaj) [inline src/middlewared/middlewared/etc_files/ctld.py:215]: Just to confirm, is this not providing all the disks ?

Comment (yocalebo) [reply src/middlewared/middlewared/etc_files/ctld.py:215]: It is but not really what the question is in reference to?

Comment (yocalebo) [reply src/middlewared/middlewared/etc_files/ctld.py:215]: Oh I see, you're talking about sharing `multipath` raw disks

Comment (yocalebo) [reply src/middlewared/middlewared/etc_files/ctld.py:215]: Hmm, no just kidding. Not sure what you're asking? 😄

**Pull request merged**